### PR TITLE
[Method References] LPS-161155 Replace method setServletContext with field _servletContext in class extends BaseJSPSettingsConfigurationAction

### PR DESCRIPTION
**Pull request by W2jx**
Hi @tinatian @dantewang , I replace method setServletContext with field _servletContext in class extends BaseJSPSettingsConfigurationAction in this pr.

### CODE REVIEW

**Comment by W2jx** [comment]
Hi @catarineaguiar , Could you please help me check the test results when they finish ? Thanks!

File: portal-kernel/src/com/liferay/portal/kernel/portlet/BaseJSPSettingsConfigurationAction.java [hunk 1]
```diff
@@ -90,15 +90,13 @@ public void include(
...
-	public void setServletContext(ServletContext servletContext) {
-		_servletContext = servletContext;
-	}
-
 	protected ServletContext getServletContext(
 		HttpServletRequest httpServletRequest) {
 
-		if (_servletContext != null) {
-			return _servletContext;
+		ServletContext servletContext = getServletContext();
```
Comment (dantewang) [inline portal-kernel/src/com/liferay/portal/kernel/portlet/BaseJSPSettingsConfigurationAction.java:96]: @W2jx This one is a bit special. The `getServletContext` method comes from `LiferayPortlet` and it has default logic. Could you double check whether the default logic can return the same servlet context you referenced?

Comment (W2jx) [reply portal-kernel/src/com/liferay/portal/kernel/portlet/BaseJSPSettingsConfigurationAction.java:96]: Ok, I will double check it.

**Comment by tinatian** [comment]
@W2jx It's been 5 days and no response, closed, please resend.

**Pull request closed without merging**